app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,26 @@
+<li>
+  <%= comment.content %>
+  <br>
+  <% user = comment.user %>
+  by
+  <%= link_to user.name, user %>
+  <%= time_ago_in_words(comment.created_at) %> ago.
+  <br>
+  <br>
+  <%= form_for @comment, url: comments_path do |f| %>
+    <%= render 'shared/comment_errors', comment: @comment %>
+    <%= f.text_area :content, placeholder: "Add a reply", cols: 30, rows: 5 %>
+    <%= f.hidden_field :comment_id, value: comment.id %>
+    <%= f.hidden_field :post_id, value: @post.id %>
+    <%= f.hidden_field :user_id, value: current_user.id %>
```

Comment:
Hmm, yeah I remember reading about security issues with hidden_fields. I believe the only thing that I need to be worrying about is the user id, because the comment and post id's can be arbitrary and unharmful.